xml/System.Net.PeerToPeer.Collaboration/PeerApplication.xml

```diff
@@ -484,7 +484,7 @@
         <param name="objA">The first <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> to compare.</param>
         <param name="objB">The second <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> to compare.</param>
         <summary>Determines whether the specified <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> objects are considered equal.</summary>
-        <returns>Returns <see langword="true" /> if the specified objects have the same globally unique application <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Id" /> and <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Description" />, else <see langword="false" />.</returns>
+        <returns><see langword="true" /> if the specified objects have the same globally unique application <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Id" /> and <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Description" />, else <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the specified objects have the same globally unique application <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Id" /> and <see cref="P:System.Net.PeerToPeer.Collaboration.PeerApplication.Description" />; otherwise, <see langword="false" />.</returns>
```